### PR TITLE
Update experience names and order to match portal 

### LINKED ISSUE #415: Rename CosmosDB apis to be consistent with the portal
This is what the portal uses:
37675581-d4b0f5e6-2c32-11e8-85da-b75afce5ef9f

This is what we use:
screen shot 2018-03-20 at 2 13 32 pm

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Fixes #415

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
image

image

File: src/utils/experienceNames.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* ------------------------------------------------------------------------------------------
+ *   Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { Experience } from "../constants";
+
+export function getExperienceName(experience: Experience) {
+    switch (experience) {
+        case Experience.DocumentDB: return "SQL";
```
[the remark this answers is not in the document]
Comment (StephenWeatherford) [reply src/utils/experienceNames.ts]: Yes, but I don't know what affect it has on the portal. These constants are passed straight to Azure. The template files created still use the old "DocumentDB" and "Graph" values.

**Comment by EricJizbaMSFT** [comment]
@fiveisprime's comment:
> Let's just go with "SQL" and we can link to the FAQ in the changelog for clarification. This will be consistent with the portal.

IMO the portal is not something we want to be consistent with in this case. I think we were all VERY confused for a long time about this change. I vote we put "SQL (DocumentDB)". We can remove "(DocumentDB)" once we're confident people know what the change is

**Comment by StephenWeatherford** [comment]
@fiveisprime @EricJizbaMSFT Personally I agree with Eric on the DocumentDB issue.  What if I add it just to the description when you're creating it (and don't show it anywhere else)?  That will put it in a smaller font.

File: src/tree/AttachedAccountsTreeItem.ts [hunk 2]
```diff
@@ -124,7 +124,7 @@ export class AttachedAccountsTreeItem implements IAzureParentTreeItem {
 
     public async attachEmulator(): Promise<void> {
         let connectionString: string;
-        const defaultExperience = <Experience>await vscode.window.showQuickPick(['MongoDB', 'DocumentDB'], { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
+        const defaultExperience = <Experience>await vscode.window.showQuickPick(['MongoDB', 'SQL'], { placeHolder: "Select a Database Account API...", ignoreFocusOut: true });
```
[the remark this answers is not in the document]
Comment (StephenWeatherford) [reply src/tree/AttachedAccountsTreeItem.ts]: Done.

**Comment by StephenWeatherford** [comment]
@EricJizbaMSFT @PrashanthCorp What do you think of this refactoring?  The usage of API vs experience is inconsistent in Azure, but this seems like a reasonable way to interpret it.

**Comment by PrashanthCorp** [comment]
Closing this.

**Pull request merged**